src/experimental.js

```diff
@@ -0,0 +1,106 @@
+// @flow
+import Config from "react-native-config";
+import { AsyncStorage } from "react-native";
+import {
+  setEnvUnsafe,
+  isEnvDefault,
+  changes,
+} from "@ledgerhq/live-common/lib/env";
+import type { EnvName } from "@ledgerhq/live-common/lib/env";
+
+import logger from "./logger";
+
+export type FeatureCommon = {
+  name: EnvName,
+  title: string,
+  description: string,
+  shadow?: boolean,
+};
+
+export type FeatureToggle = {
+  type: "toggle",
+  valueOn?: any,
+  valueOff?: any,
+};
+
+export type Feature = FeatureCommon & FeatureToggle;
+
+export const experimentalFeatures: Feature[] = [
+  {
+    type: "toggle",
+    name: "MANAGER_DEV_MODE",
+    title: "Dev mode",
+    description: "enables developer apps in manager",
+  },
+  // {
+  //   type: "toggle",
+  //   name: "EXPERIMENTAL_USB",
+  //   title: "Experimental USB",
+  //   description:
+  //     "Alternative USB implementation that might help solving USB issues. Enabling this feature might create UI glitches.",
+  // },
+  // {
+  //   shadow: true,
+  //   type: "toggle",
+  //   name: "EXPERIMENTAL_EXPLORERS",
+  //   title: "Experimental explorers",
+  //   description: "switch to the new version of explorers",
+  // },
+  // {
+  //   shadow: true,
+  //   type: "toggle",
+  //   name: "FORCE_PROVIDER",
+  //   valueOn: 4,
+  //   valueOff: 1,
+  //   title: "Manager test app provider=4",
+  //   description: "enables yet `unreleased` apps in manager",
+  // },
```

Comment:
since you still have the shadow system, i guess you can uncomment it ;) i think it still makes sense when we do custom build to have this feature ;)